#!/usr/bin/env python
PACKAGE = "pfvtr"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("velocity_gain",    double_t,    0, "Velocity Gain", 1,  0, 5)
gen.add("turn_gain", double_t, 0, "Turn Gain angle/pixel",    1, 0,   200)
gen.add("use_uncertainty",   bool_t,   0, "Use uncertainty",  True)

exit(gen.generate(PACKAGE, "pfvtr", "Controller"))
